tests/test_utils.py

```diff
@@ -9,10 +9,14 @@
 """REANA-DB utils tests."""
 
 from __future__ import absolute_import, print_function
+from reana_commons.config import SHARED_VOLUME_PATH
+import os
```

Comment:
We have a convention to separate future statements, python modules and custom modules with an empty line, so in this case it could be:
```python
from __future__ import absolute_import, print_function

import os

from reana_commons.config import SHARED_VOLUME_PATH
```